appeals/api/src/server/tests/integrations/mocks.js

```diff
@@ -1,134 +1,109 @@
-				neighbouringSiteSafetyDetails: null
-			},
-			{
-				neighbouringSiteAddressLine1: 'veniam est ut ea amet',
-				neighbouringSiteAddressLine2: null,
-				neighbouringSiteAddressTown: 'tempor exercitation mollit eiusmod in',
-				neighbouringSiteAddressCounty: null,
-				neighbouringSiteAddressPostcode: 'deserunt amet aute',
-				neighbouringSiteAccessDetails: 'voluptate veniam consequat',
-				neighbouringSiteSafetyDetails: null
-			},
-			{
-				neighbouringSiteAddressLine1: 'in',
-				neighbouringSiteAddressLine2: 'dolore minim',
-				neighbouringSiteAddressTown: 'sunt ut',
-				neighbouringSiteAddressCounty: 'reprehenderit Ut dolore',
-				neighbouringSiteAddressPostcode: 'dolor',
-				neighbouringSiteAccessDetails: 'nulla adipisicing',
-				neighbouringSiteSafetyDetails: null
-			}
-		],
-		appellantCostsAppliedFor: false
+		lpaCode: 'Q9999',
+		nearbyCaseReferences: ['1000000'],
+		neighbouringSiteAddresses: [],
+		originalDevelopmentDescription: 'A test description',
+		ownersInformed: true,
+		ownsAllLand: true,
+		ownsSomeLand: true,
+		siteAccessDetails: ['Come and see'],
+		siteAddressCounty: 'Somewhere',
+		siteAddressLine1: 'Somewhere',
+		siteAddressLine2: 'Somewhere St',
+		siteAddressPostcode: 'SOM3 W3R',
+		siteAddressTown: 'Somewhereville',
+		siteAreaSquareMetres: 22,
+		siteSafetyDetails: ["It's dangerous"]
 	},
 	documents: [
 		{
-			documentId: '4547fec8-16b2-47bb-836d-4d0baac04079',
-			filename: 'exercitation reprehenderit laborum eu enim',
-			originalFilename: 'cupidatat',
-			size: 23810727,
-			mime: 'non cupidatat ea',
-			documentURI: 'elit est Excepteur cupidatat',
-			dateCreated: '1890-01-01T22:21:50.0Z',
-			documentType: 'originalApplicationForm'
+			dateCreated: '2024-03-01T13:48:35.847Z',
+			documentId: '001',
+			documentType: 'appellantCostsApplication',
+			documentURI:
+				'https://pinsstdocsdevukw001.blob.core.windows.net/uploads/055c2c5a-a540-4cd6-a51a-5cfd2ddc16bf/788b8a15-d392-4986-ac23-57be2f824f9c/--12345678---chrishprofilepic.jpeg',
+			filename: 'img.jpg',
+			mime: 'image/jpeg',
+			originalFilename: 'oimg.jpg',
+			size: 10293
 		}
 	],
 	users: [
 		{
+			emailAddress: 'test@test.com',
+			firstName: 'Testy',
+			lastName: 'McTest',
 			salutation: null,
-			firstName: 'Paul',
-			lastName: 'Pogba',
-			emailAddress: null,
 			serviceUserType: 'Appellant'
-		},
-		{
-			salutation: null,
-			firstName: 'Ray',
-			lastName: 'Liotta',
-			emailAddress: 'test@example.com',
-			serviceUserType: 'Agent'
 		}
 	]
 };
 
 export const validLpaQuestionnaire = {
-	questionnaire: {
-		caseReference: '6000526',
-		LPACode: 'Q9999',
-		isAppealTypeAppropriate: true,
-		doesTheDevelopmentAffectTheSettingOfAListedBuilding: true,
-		affectedListedBuildings: [
+	casedata: {
+		caseReference: '6000000',
+		lpaQuestionnaireSubmittedDate: new Date(2024, 5, 1).toISOString(),
+		lpaStatement: 'cupidatat ipsum eu culpa',
+		siteAccessDetails: ['Here it is'],
```

Comment:
It's an array now?

Reply:
yes, comes and goes as an array, but there will only be 2 items (the appellant and lpa answers)